instructions.md

```diff
@@ -0,0 +1,352 @@
+# Using the MASST+ binary
+
+The `masst_plus` binary integrates all tools needed to index and query mass spectrum database. It is designed to be fast and memory-efficient.
+
+The tool comes with two types of search: exact and error-tolerant (analog). Separate indices are needed for each search type. This tool is thus split into two subcommands: exact and analog, each of which has its own set of subcommands for creating, adding to, and querying indices.
+
+You can always consult `masst_plus --help` for detailed description of the available commands and flags. 
+
+## The search types
+
+**Exact search** mode matches query spectra against database spectra based on shared peaks only. The index subcommand ingests files into the disk-based database index. 
+
+**Analog search** mode (also known as error-tolerant) matches query spectra against database spectra based on sahred and shifted peaks. The index subcommand ingests files into the disk-based database index.
```

Comment:
combine with prev paragraph.

Have one sentecne for indexing, explaining a bit more that it pre-processes spectra into a format that's fast to query. For error tolerant explain taht the outputs will be pairs of spectra where one is shifted by some delta